### PR TITLE
[CTX-608] fix: iac-test debug namespaces

### PR DESCRIPTION
**Pull request by craigfurman**
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

#### What does this PR do?

These logs weren't showing up when `-d` was passed. Also bump snyk-iac-test.

#### What are the relevant tickets?

https://snyksec.atlassian.net/browse/CTX-608

### CODE REVIEW

File: src/lib/iac/test/v2/local-cache/policy-engine/constants/utils.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as os from 'os';
 
+// TODO: update!
```
Comment (craigfurman) [inline src/lib/iac/test/v2/local-cache/policy-engine/constants/utils.ts:3]: Waiting on https://github.com/snyk/snyk-iac-test/pull/199 to release before I update these checksums. Looking for approval in advance assuming that's non-controversial?

Comment (craigfurman) [reply src/lib/iac/test/v2/local-cache/policy-engine/constants/utils.ts:3]: This is now done, but it looks like the latest release is no longer producing windows/arm64 binaries of snyk-iac-test: https://app.circleci.com/pipelines/github/snyk/snyk-iac-test/1161/workflows/2e568764-52be-46fc-8864-e573d6f179b7/jobs/5975

> DEPRECATED: skipped windows/arm64 build on Go < 1.17 for compatibility, check https://goreleaser.com/deprecations/#builds-for-windowsarm64 for more info

We should figure out if this is something we want, and maybe fix it before making any further snyk-iac-test changes 🤔 

WDYT codeowners?

Comment (craigfurman) [reply src/lib/iac/test/v2/local-cache/policy-engine/constants/utils.ts:3]: Although it looks like that build got dropped here: https://github.com/snyk/cli/commit/dc8428d6d3a1a87e08eb32a95aa3e0c196d91941

Comment (craigfurman) [reply src/lib/iac/test/v2/local-cache/policy-engine/constants/utils.ts:3]: Looks like this is known: https://snyk.slack.com/archives/C02JMMTLUF9/p1686669461907859

**Review by andreeaneata — APPROVED** [review]

**Pull request merged**